vpnglean/CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Android")
     set(OPENSSL_LIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../3rdparty/openSSL/static/lib/${OPENSSL_ARCH})
     set(OPENSSL_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../3rdparty/openSSL/static/include)
     set(CARGO_CMD OPENSSL_LIB_DIR=${OPENSSL_LIB_DIR} OPENSSL_INCLUDE_DIR=${OPENSSL_INCLUDE_DIR} AR=llvm-ar ${CARGO_EXECUTABLE} build)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
+    set(CARGO_CMD LIBRARY_PATH="$DEVELOPER_SDK_DIR/MacOSX.sdk/usr/lib:$LIBRARY_PATH:-" ${CARGO_EXECUTABLE} build)
```

Comment:
Totally stole this from https://github.com/mozilla/glean/blob/b2418780195ae3daa200b9cffdb61c6163e0671b/build-scripts/xc-cargo.sh#L9

Reply:
Note that I recently had to fix this up in another place for more recent Rust: https://github.com/mozilla/glean/pull/2256